src/main/java/org/cbioportal/model/SampleClinicalDataCollection.java

```diff
@@ -0,0 +1,19 @@
+package org.cbioportal.model;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class SampleClinicalDataCollection {
```

Comment:
Probably should start making Models not returned by MyBatis (Java Objs created by query results... Persistence layer) Immutable.

I would suggest making the class immutable (using a Builder Pattern to build the object)